autoconnect/autoconnect-settings/src/lib.rs

```diff
@@ -99,10 +97,11 @@ pub struct Settings {
     /// Maximum allowed number of backlogged messages. Exceeding this number will
     /// trigger a user reset because the user may have been offline way too long.
     pub msg_limit: u32,
-    /// Maximum number of pending notifications for individual UserAgent handlers.
-    /// (if a given [autoconnect-common::RegisteredClient] receives more than this number, the calling
-    /// thread will lock.)
-    pub max_pending_notification_queue: u32,
+    /// Sets the maximum number of concurrent connections per actix-web worker.
+    ///
+    /// All socket listeners will stop accepting connections when this limit is
+    /// reached for each worker.
+    pub actix_max_connections: Option<usize>,
```

Comment:
Let's add a setting and support for actix server [workers](https://docs.rs/actix-server/latest/actix_server/struct.ServerBuilder.html#method.workers) as well, since I don't know if the kubernetes core limit is sufficiently differentiated from whatever rust is using to read the core count. (`grep processor /proc/cpuinfo | wc -l`?)

At least that way, we can know for sure.

Reply:
it's all based on `available_parallelism` so we can look at the log statement when this goes out. Added workers